### ISSUE TITLE
Feature: ios:accessibilityPerformEscape()

### ISSUE THREAD

**Issue by m-abs** [Notalib/nativescript-accessibility-ext#6]
**accessibilityPerformEscape()**

Dismisses a modal view and returns the success or failure of the action.

*User action:* A two-finger Z-shaped gesture that dismisses a modal dialog, or goes back one level in a navigation hierarchy. Returns true, if we're handling the escape. Dev note: Unlike accessibilityPerformMagicTap() this cannot be added to the AppDelegate in NativeScript.
